From 99bba8d47db64a9346499e721321e294250df248 Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Thu, 1 Aug 2013 13:08:54 +0800
Subject: [PATCH 4/5] ENGR00273437 use Gallery instead of Gallery2 on mx6sl

use Gallery instead of Gallery2 on mx6sl, because
Gallery2 need OpenGL ES 2.0 API else it will report
error but mx6sl no GPU 3D core. So if the chip has
GPU3D it will use Gallery2, else use Gallery.

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Android.mk b/Android.mk
index cf5b767..b2c6654 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,3 +1,4 @@
+ifeq ($(HAVE_FSL_IMX_GPU3D),true)
 LOCAL_PATH:= $(call my-dir)
 
 include $(CLEAR_VARS)
@@ -54,3 +55,4 @@ ifeq ($(strip $(LOCAL_PACKAGE_OVERRIDES)),)
 include $(call all-makefiles-under, $(LOCAL_PATH))
 
 endif
+endif
-- 
1.8.0

